db/migrate/20170221093606_update_old_redirect_unpublishings.rb

```diff
@@ -0,0 +1,24 @@
+class UpdateOldRedirectUnpublishings < ActiveRecord::Migration[5.0]
+  def up
+    Unpublishing
+      .where(type: :redirect)
+      .where(redirects: nil)
+      .find_each do |unpublishing|
+        # this magical line works because the 'redirects' getter is overriden
+        # to generate a JSON blob automatically if it is nil within the
+        # database
+        unpublishing.update_attribute(:redirects, unpublishing.redirects)
+      end
+  end
+
+  def down
```

Comment:
True, it doesn't do the exact reverse. The `down` method here is more of a safety net in case we needed to rollback the code and any new unpublishings made while the new code was active, wouldn't have an `alternative_path` set.